hacking/env-setup.fish

```diff
@@ -65,8 +65,9 @@ set -gx ANSIBLE_LIBRARY $ANSIBLE_HOME/library
 
 # Do the work in a fuction
 function gen_egg_info
-
-    if test -e $PREFIX_PYTHONPATH/ansible*.egg-info
+    # Cannot use `test` on wildcards. 
+    # @see https://github.com/fish-shell/fish-shell/issues/5960
+    if count $PREFIX_PYTHONPATH/ansible*.egg-info > /dev/null
         rm -rf "$PREFIX_PYTHONPATH/ansible*.egg-info"
```

Comment:
Ummm... this bit won't work, because the "*" is quoted?

Reply:
Hrm, I tested it locally and it worked nicely. Are you saying I should quote it?

Reply:
Noooo.

The `rm -rf` argument should *not* be quoted, is what I'm saying.

Reply:
`rm -rf "$PREFIX_PYTHONPATH/ansible*.egg-info"` **does not work**.

When you say "it worked nicely", what you mean is presumably that you're not seeing the error anymore. But that's probably just because no file matches, so the `rm` is never executed.

Reply:
Oh, I see now. I was still looking at the `if` line, doh. I'll fix and test locally.

Reply:
Nice catch! Fixed, tested, pushed!